apps/rule-manager/client/src/ts/components/helpers/updateRule.ts

```diff
@@ -0,0 +1,14 @@
+import { FormDataForApiEndpoint } from "./createRule";
+
+export const updateRule = async (ruleForm: FormDataForApiEndpoint) => {
+    // We would always expect the ruleForm to include an ID when updating a rule
+    if (!ruleForm.id) return (new Response(null, {status: 400, statusText: "Update endpoint requires a rule ID"}))
```

Comment:
Related to the comment above about status checking, I don't think the interface for this should be a `Response` – coercing a synchronous validation error into that shape w/ a status code is a bit odd, it might make someone think a server was involved.